src/template.rs

```diff
@@ -316,8 +319,7 @@ pub fn render_string_gracefully(
 
 fn print_files_with_errors_warning(files_with_errors: Vec<(String, liquid_core::Error)>) {
     let mut msg = format!(
-        "\n{} {}",
-        emoji::WARN,
+        "\n{}",
```

Comment:
I believe this does now behave unintended.

Before the change it was printed a `\n<warn-emoji> <message>`
Now it's going to be `<warn-emoji>\n<message>`

suggestion: removing the `\n` completely here and therefore logging an `info!("\n")` at just before line 336. That should lead to the same behavior as before.